### PR TITLE
Fixes #11 Contact Us Page + Form having basic validations

### LINKED ISSUE #11: Design Contact Us form
Suggestion as in issue #1.

### PR DESCRIPTION
**Pull request by akshitdandyan**
- Contact us form with minimalistic and simple UI
- Currently showing demo submitted message, can modify later if we integrate backend

### CODE REVIEW

**Comment by princiya** [comment]
@akshitdandyan thanks for the PR.
but we already have a PR in progress #21 by @sid-khuntwal. also the corresponding #11 issue has been claimed.
when you look for contributions, please discuss upfront or claim the issue to avoid duplicate work.
feel free to propose new issues after reviewing the conversations from the existing issues and PRs.
best of luck.

File: src/components/ContactUs/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    // modifying label colors when users leaves any field empty
+    const modifyLabelColor = ( labelIndex ) => {
+        const newStyles = labelStyles.map( ( eachStyle, i) => {
+            if( i === labelIndex ) return eachStyle = { bottom: "28px", opacity: 1, color: "red" };
+            return eachStyle;
+        });
+        return setLabelStyles( newStyles );
+    }
+
+    const submitForm = (e) => {
```
Comment (princiya) [inline src/components/ContactUs/ContactUs.jsx:44]: @sid-khuntwal is not working on form validation. so when his PR is merged, you can definitely add the validation logic in a follow up PR.

Comment (princiya) [reply src/components/ContactUs/ContactUs.jsx:44]: based on the slack study group members, this will be frontend only for now.

Comment (akshitdandyan) [reply src/components/ContactUs/ContactUs.jsx:44]: Alright :)

**Comment by princiya** [comment]
@akshitdandyan #30 has been merged. If you like, you can now either modify this PR or open a new PR to add form validation.

**Comment by akshitdandyan** [comment]
@princiya I have opened a pull request for the same, please assign it to me. And also I opened a pull request regarding PWA support, if you like to add that feature then I would like to work on it.

**Pull request closed without merging**